### ISSUE TITLE
Unused using references in nodes

### ISSUE THREAD

**Issue by smakhtin** [elliotwoods/VVVV.Packs.Image#6]
Currently library code full of unused references, i can clean it, but it can be hard to merge, if you changing something. If not, you can wait the pull request from me in a day or two.

**Comment by elliotwoods** [comment]
can you have a look after the Beta is released?
if you let me know when you'll be looking i can be hands off during that time.

**Comment by smakhtin** [comment]
Yep, sure. I can also redesign some interfaces for better C#ish style.

**Comment by elliotwoods** [comment]
just make sure it's in a seperate pull request of course ;)
